src/Project.js

```diff
@@ -1,19 +1,31 @@
-import React, { Component } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
+
 import './Project.css';
 
-class Project extends Component {
-  constructor(props) {
-    super();
-    this.match = props.match;
-  }
+const Project = (props) => {
```

Comment:
If all you have is render function, you can simplify it into what is called a "functional component", it only takes props and doesn't have any other logic than what it needs to show.  The airbnb eslint config will yell at you to get rid of the class if you don't need it.